src/utilities/focus.ts

```diff
@@ -130,6 +130,7 @@ export function isElementTabbable(element: HTMLElement): boolean {
     (element.tagName === 'A' ||
       (element.tagName === 'BUTTON' && !(element as HTMLButtonElement).disabled) ||
```

Comment:
That doesn't work in typescript. :) But the static analysis works, which is why we need to cast so it stops complaining.